package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@heroicons/react": "^2.0.18",
-    "@storyblok/react": "^2.4.7",
+    "@storyblok/react": "^3.0.2",
```

Comment:
<sarcasm>My favorite kind of release notes. Major change. Warning warning. And nothing else... 😠 </sarcasm>

https://github.com/storyblok/storyblok-react/releases/tag/v3.0.0

Reply:
Yikes...Storyblok 😬